pom.xml

```diff
@@ -101,10 +101,11 @@
     <version.zipkin>1.13.1</version.zipkin>
 
     <!-- OpenTracing related versions -->
-    <version.opentracing>0.22.0</version.opentracing>
+    <version.opentracing>0.30.0</version.opentracing>
     <version.opentracing.tracerresolver>0.1.0</version.opentracing.tracerresolver>
-    <version.opentracing.servlet>0.0.8</version.opentracing.servlet>
+    <version.opentracing.servlet>0.0.9</version.opentracing.servlet>
     <version.opentracing.spanmanager>0.0.5</version.opentracing.spanmanager>
+    <version.jaeger>0.20.0</version.jaeger>
```

Comment:
such as here, you can define `version.jaeger.commons-codec` to reference from the above `module.xml`